src/main/java/io/mosip/print/service/impl/PrintServiceImpl.java

```diff
@@ -496,6 +493,8 @@ private String setTemplateForMaskedUIN(String cardType, String uin, String vid,
 	 */
 	private boolean setQrCode(String qrString, Map<String, Object> attributes)
 			throws QrcodeGenerationException, IOException {
+        String pincode = "1234";
+        String photob64 = (String)attributes.get("ApplicantPhoto");
```

Comment:
Will this always be the same field name?

Reply:
These are the same field names as used by MOSIP in their own mapping of the json so I find these field name values to be stable.

Reply:
Would it be hard to use the mapping instead? Those could potentially change.